docs/docs/artifact-store.md

```diff
@@ -250,22 +247,17 @@ Transfer-Encoding: chunked
 
 ### Automatic Path creation
  
-The path in the asset store is computed that way:
-
-* HEAD Request to the asset
-* if ETag is available, take ETag HTTP Header
-* if ETag is not available (dumb HTTP server), download the resource and compute the Sha-1 hash manually
-* filename of the url remains the same
+The path in the asset store is computed in the following way:
 
-The complete path is {artifact store base}/{ETag or ContentHash}/{filename of asset}
-This effectively will create a path, that is unique to the content of the resource.
-If the same URL holds a different entity, a new path is created.
-The same path is downloaded only once. 
-In other words if the path is available in the artifact store, it is not downloaded again.
+1. HEAD Request to the asset.
+2. If ETag is available, take ETag HTTP header.
+3. If ETag is not available, download the resource and compute the Sha-1 hash manually.
+4. The filename of the url remains the same.
```

Comment:
Each URL points to a file. Since the URL is replaced with a URI, we make sure, the filename stays the same.